torch/_dynamo/output_graph.py

```diff
@@ -1138,7 +1140,8 @@ def remove_unused(node):
                     fake = (
                         arg.fake_tensor if arg.fake_tensor is not None else arg.example
                     )
-                    used_symbols |= free_symbols(fake)
+                    if arg.has_symbols:
```

Comment:
How come? If it doesn't have free symbols, free_symbols returns empty set which is fine too

Reply:
I think invoking free_symbols on a function type doesn't work :( this is to circumvent the other types that can now flow through here. Much sadness, really.

Reply:
`free_symbols` is supposed to work for everything that is an argument type. So if we add new argument types, we need to teach it to ignore it.

But this leads to the other thing I'm pretty unsure about, which is I don't... really want these functions as inputs to the graph.